osu.Game/Screens/Ranking/Expanded/Statistics/PerformanceStatistic.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Threading;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Game.Scoring;
+
+namespace osu.Game.Screens.Ranking.Expanded.Statistics
+{
+    public class PerformanceStatistic : CounterStatistic
+    {
+        private readonly ScoreInfo score;
+
+        private readonly Bindable<int> performance = new Bindable<int>();
+
+        private readonly CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
+
+        public PerformanceStatistic(ScoreInfo score)
+            : base("PP", 0)
+        {
+            this.score = score;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(ScorePerformanceManager performanceManager)
+        {
+            if (score.PP.HasValue)
+            {
+                performance.Value = (int)score.PP.Value;
```

Comment:
not sure about this cast - maybe it should be a round instead, to prevent truncation

Reply:
I'm not sure we're applying any rounding everywhere else in the codebase where we are manipulating pp values, so I would say that we should truncate for now for consistency with the rest of the codebase, no?

Reply:
Actually most places that I can see do `ToString()` from decimal, which rounds.